src/environments/environment.ts

```diff
@@ -12,6 +12,7 @@ export const environment: EnvironmentConfig = {
   },
   appVersion: APPLICATION_VERSION,
   appRelease: APPLICATION_RELEASE,
+  stage: '',
```

Comment:
See comment above - when this is optional, you can avoid specifying this here, which is cleaner.